docs/controlshift.rst

```diff
@@ -0,0 +1,42 @@
+Controlshift
+=========
+
+********
+Overview
+********
+
+Controlshift is a platform for creating campaigns with distributed events, local groups, and member-led petitions. This class allows you to interact with select functions of the the `Controlshift Authenticated REST API <https://developers.controlshiftlabs.com/#authenticated-rest-api/>`_ .
```

Comment:
In the docs, I'm unsure whether we most frequently use the word "class" or if we prefer "connector" for general descriptions. This might merit a small change, but it's up to your judgment. The "class" language a couple lines below is certainly correct, though, since it refers to the technical specifics of instantiation.